pycbc/waveform/ringdown.py

```diff
@@ -29,6 +29,7 @@
 import lalsimulation as lalsim
 from pycbc.types import TimeSeries, FrequencySeries, float64, complex128, zeros
 from pycbc.waveform.waveform import get_obj_attrs
+from pycbc.conversions import get_lm_f0tau
```

Comment:
If this function is only going to be in the conversions module now, then the function `get_lm_f0tau_allmodes` should also be in conversions, or this function could be generalized to have the ability to return various (l, m) modes... This is not pressing though, it could be done in a separate PR later, but if that's the case we should open an issue so that we don't forget to do that in the future.